core/txAdmin.js

```diff
@@ -206,7 +206,9 @@ export default class TxAdmin {
         }
 
         //Once they all finish loading, the function below will print the banner
-        printBanner();
+        if (!process.env.SKIP_BANNER) {
```

Comment:
After some thought, it would probably make more sense to have this be derived from a convar launch arg to FXServer, rather than through an env var. Thereby matching current conventions in regards to configurable startup options. 

Like the following

https://github.com/tabarra/txAdmin/blob/e956d84b14e523e7c7165905d79b4743472c5b83/core/globalData.ts#L212-L226